docs/topics/forms/descriptions/connect-description.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Connect the description
+layout: default
+parent: Input and description
+nav_order: 2
+---
+
+# Connect the description to the form field
+
+For screen reader users, it's important that the description is announced along with the form field. By connecting the description to the form field using `aria-describedby`, the label and description are announced together when a screen reader user focuses on the form field.
+
+## Why is this important?
+
+Screen readers, such as JAWS, switch to "forms mode" when processing content within a `<form>` element. In this mode, screen readers only read the form fields, including any associated information (for example with `aria-describedby`). Unassociated information is not read aloud unless the user searches for it.
+
+## The aria-describedby/id relationship
+
+The HTML basics:
+
+- Give the description a unique ID: `id="description-name"`.
+- In the form field, refer to that ID: `aria-describedby="description-name"`.
+
+```html
+<label for="fullname">Full name</label>
+<p id="description-fullname">Please fill out your first and last name.</p>
+<input 
+        id="fullname" 
+        aria-describedby="description-fullname" 
+        autocomplete="name" />
+```
+
+{: .alert }
+The IDs used must be unique for the page; otherwise, the wrong description may be announced with a form field.
+
+It's possible to add multiple descriptions to one form field. You can for example combine the text of description and an error message by separation the IDs with a space. The order of the IDs given in the `aria-describedby` attribute is the order in which they are announced.
```

Comment:
Changes: delete "for example" in second sentence, add 'the' to 'the description'; grammar fix separation > separating.

It's possible to add multiple descriptions to one form field. You can combine the text of the description and an error message by separating the IDs with a space. The order of the IDs given in the `aria-describedby` attribute is the order in which they are announced.